ssl/quic/quic_rx_depack.c

```diff
@@ -333,8 +333,7 @@ static int depack_do_frame_max_data(PACKET *pkt, QUIC_CONNECTION *connection,
     /* This frame makes the packet ACK eliciting */
     ackm_data->is_ack_eliciting = 1;
 
-    /* TODO(QUIC): ADD CODE to send |max_data| to flow control */
-
+    /* No-op - informative/debugging frame. */
```

Comment:
Yes, this is correct. Connection-level flow control can impose a limit less than the sum of the available credit for all streams, otherwise there would be little point to having a separate connection-level mechanism.

The `BLOCKED` frames are documented in the RFCs as being essentially for debugging. An implementation could use them as a performance hint but MUST NOT rely on them being generated, and need not use them.